tslint.json

```diff
@@ -1,83 +0,0 @@
-{
```

Comment:
were all the rules from ts lint moved to es lint? Because the es lint files seemed quite empty in comparison to the ts lint files

Reply:
It is empty, because of a discussion is needed about that. The reason is that some rules are not present in ESLint, some have different name and there are new rules too.

Reply:
does an issue for this exist?

Reply:
I would like to resolve ES Lint rules within issue [NAE-1463].